.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -40,129 +40,23 @@ body:
-      - "Component: io-java-gcp"
-      - "Component: io-java-hadoop-file-system"
-      - "Component: io-java-hadoop-format"
-      - "Component: io-java-hbase"
-      - "Component: io-java-hcatalog"
-      - "Component: io-java-healthcare"
-      - "Component: io-java-influxdb"
-      - "Component: io-java-jdbc"
-      - "Component: io-java-jms"
-      - "Component: io-java-kafka"
-      - "Component: io-java-kinesis"
-      - "Component: io-java-kudu"
-      - "Component: io-java-mongodb"
-      - "Component: io-java-mqtt"
-      - "Component: io-java-parquet"
-      - "Component: io-java-pulsar"
-      - "Component: io-java-rabbitmq"
-      - "Component: io-java-redis"
-      - "Component: io-java-snowflake"
-      - "Component: io-java-solr"
-      - "Component: io-java-splunk"
-      - "Component: io-java-text"
-      - "Component: io-java-tfrecord"
-      - "Component: io-java-tika"
-      - "Component: io-java-utilities"
-      - "Component: io-java-xml"
-      - "Component: io-py-avro"
-      - "Component: io-py-aws"
-      - "Component: io-py-common"
-      - "Component: io-py-files"
-      - "Component: io-py-gcp"
-      - "Component: io-py-hadoop"
-      - "Component: io-py-ideas"
-      - "Component: io-py-kafka"
-      - "Component: io-py-mongodb"
-      - "Component: io-py-parquet"
-      - "Component: io-py-tfrecord"
-      - "Component: io-py-vcf"
-      - "Component: java-fn-execution"
-      - "Component: katas"
-      - "Component: project-management"
-      - "Component: release"
-      - "Component: run-inference"
-      - "Component: runner-apex"
-      - "Component: runner-core"
-      - "Component: runner-dataflow"
-      - "Component: runner-direct"
-      - "Component: runner-flink"
-      - "Component: runner-gearpump"
-      - "Component: runner-ideas"
-      - "Component: runner-jet"
-      - "Component: runner-jstorm"
-      - "Component: runner-mapreduce"
-      - "Component: runner-py-direct"
-      - "Component: runner-py-interactive"
-      - "Component: runner-ray"
-      - "Component: runner-samza"
-      - "Component: runner-spark"
-      - "Component: runner-tez"
-      - "Component: runner-twister2"
-      - "Component: runner-universal"
-      - "Component: sdk-go"
-      - "Component: sdk-ideas"
-      - "Component: sdk-java-core"
-      - "Component: sdk-java-harness"
-      - "Component: sdk-py-core"
-      - "Component: sdk-py-harness"
-      - "Component: sdk-typescript"
-      - "Component: test-failures"
-      - "Component: testing"
-      - "Component: testing-nexmark"
-      - "Component: website"
-    validations:
-      required: true
+       - Python
+       - Java
+       - Go
+       - Typescript
+       - IO connector
+       - Beam examples
+       - Beam playground
+       - Beam katas
+       - Website
+       - Spark Runner
+       - Flink Runner
+       - Samza Runner
+       - Twister2 Runner
+       - Hazelcast Jet Runner
+       - Google Cloud Dataflow Runner
```

Comment:
Its worth calling out that moving these to checkboxes will break our auto-labeling of issues. I think that's probably ok, but we should cut most of https://github.com/apache/beam/blob/master/.github/issue-rules.yml (if not all of it)